scripts/check-status

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
oh ok, I don't really mind either way, I just noticed that github was complaining about it.